netket/operator/_continuous_operator.py

```diff
@@ -78,7 +78,7 @@ def __add__(self, other):
         ):
             from netket.operator import SumOperator
 
-            return SumOperator(self, other)
+            return SumOperator(self, other, coefficients=[1.0, 1.0])
```

Comment:
with the changes I proposed below, this should now work
```suggestion
            return SumOperator(self, other)
```